crates/pallet-feeds/src/lib.rs

```diff
@@ -349,14 +361,21 @@ impl<T: Config> Call<T> {
                     None => return vec![],
                 };
                 let feed_processor = T::feed_processor(feed_processor_id);
-                let mut objects_mappings = feed_processor.object_mappings(*feed_id, object);
-                // `FeedId` is the first field in the extrinsic. `1+` corresponds to `Call::put {}`
-                // enum variant encoding.
-                objects_mappings.iter_mut().for_each(|object_mapping| {
-                    // update the offset to include the absolute offset in the extrinsic
-                    object_mapping.offset += 1 + mem::size_of::<T::FeedId>() as u32
-                });
+                let objects_mappings = feed_processor.object_mappings(*feed_id, object);
+
                 objects_mappings
+                    .into_iter()
+                    .map(|object_mapping| {
+                        CallObject {
+                            // scope the key of the object to the feed_id namespace
+                            key: crypto::sha256_hash((feed_id, object_mapping.key).encode()),
```

Comment:
This will work, but do we really need to encode it (and do another allocation)?
We can just do 2 `.update()` calls.

Reply:
I'm not aware of update(), can you point me to some rust docs pls ?

Reply:
Just go to the source of `sha256_hash` function